coding/python_wrappers_for_cpp_with_pybind11.rst

```diff
@@ -330,31 +330,25 @@ Ndarray
 ^^^^^^^
 
 The function ``ExampleOne::doSomethingWithArray`` takes an ``ndarray::Array`` argument.
-To enable automatic conversion to and from ``numpy.ndarray`` in Python add the following includes (right below the pybind11 ones):
+To enable automatic conversion to and from ``numpy.ndarray`` in Python add the following include (right below the pybind11 ones):
 
 .. code-block:: cpp
 
-    #include "numpy/arrayobject.h"
     #include "ndarray/pybind11.h"
 
-We also need to make sure the ``numpy`` module is imported by adding:
-
-.. code-block:: cpp
-
-    if (_import_array() < 0) {
-            PyErr_SetString(PyExc_ImportError, "numpy.core.multiarray failed to import");
-            return nullptr;
-    };
-
 Then the function can be wrapped as normal:
 
 .. code-block:: cpp
 
     clsExampleOne.def("doSomethingWithArray", &ExampleOne::doSomethingWithArray);
 
-.. warning::
```

Comment:
Why remove the warning?

Reply:
Because no one should be adding ``_import_array`` calls to new code anymore.